adapter-base-quarkus/src/main/java/org/eclipse/hono/adapter/quarkus/AbstractProtocolAdapterApplication.java

```diff
@@ -570,4 +584,19 @@ public KafkaClientMetricsSupport kafkaClientMetricsSupport(final KafkaMetricsOpt
             return NoopKafkaClientMetricsSupport.INSTANCE;
         }
     }
+
+    /**
+     * Creates the notification receiver.
+     *
+     * @return The bean instance.
+     */
+    public NotificationReceiver notificationReceiver() {
+        if (kafkaNotificationConfig.isConfigured()) {
+            return new KafkaBasedNotificationReceiver(vertx, kafkaNotificationConfig);
+        } else {
+            // TODO provide AMQP based notification receiver
+            return new NoOpNotificationReceiver();
```

Comment:
@b-abel Do you want to add usage of `ProtonBasedNotificationReceiver` here now (corresponding PR adding that class has been merged)? 
Otherwise we have to make sure that this gets added in a follow-up PR before the Hono 1.11.0 release.

Reply:
I'll check how much change this would bring. If it is trivial, I could add it here.

Reply:
@calohmn I would prefer to add this in a follow-up PR.